robots.txt

```diff
@@ -0,0 +1,19 @@
+User-Agent: *
+Disallow: /*/reservation/
+Disallow: /*/profil/
+Disallow: /*/cancellation/
+Disallow: /*/fh/
+Disallow: /*.pdf
+Disallow: /*.mini
+Disallow: /security-certificate/
+Disallow: /ssr*
+Disallow: /identification/
+Disallow: /express/
+Disallow: /ficheHotel/
+Disallow: /factsheet/
+Disallow: /*.do
+Disallow: /*.action
+Disallow: /*/mini.shtml
+Disallow: /newsletter/
+Disallow: /geo/
+Disallow: /errors/others/index.*.jshtml
```

Comment:
Add a sitemap directive at the end of this file that points to the sitemap-index file

Reply:
@shsteimer sitemap directive has been added